gulpfile.js

```diff
@@ -0,0 +1,313 @@
+var gulp = require('gulp'),
+    gutil = require('gulp-util'),
+    rename = require('gulp-rename'),
+    consolidate = require('gulp-consolidate'),
+    coffee = require('gulp-coffee'),
+    imagemin = require('gulp-imagemin'),
+    fontgen = require('gulp-fontgen'),
+    iconfont = require('gulp-iconfont'),
+    iconfontcss = require('gulp-iconfont-css'),
+    sass = require('gulp-sass'),
+    autoprefixer = require('gulp-autoprefixer'),
+    gulpSequence = require('gulp-sequence').use(gulp),
+    shell = require('gulp-shell'),
+    livereload = require('gulp-livereload');
+
+var config = {
+  temporaryDir: './assets',
+  assetsDir:    './web/assets'
+};
+
+var minify = true;
+
+function getStrippedPath(folderToSearch, path) {
```

Comment:
die code zit in zowel de gewone als de andere gulpfile. Dat maakt maintenance lastiger. Kan dit eventueel niet in een gulp module (of hoe dat ook genoemd wordt) gestoken worden?